sp800-63a/sec5_proofing.md

```diff
@@ -2,28 +2,30 @@
 
 <div class="breaker"></div>
 
-# <a name="ipv-section"></a> 5. Identity Resolution, Validation and Verification
+# <a name="ipv-section"></a> 5. Identity Resolution, Validation, and Verification
 
 _This section is normative._
 
-The following sections list the objectives and steps a CSP SHALL follow to identity proof an individual to meet requirements per IAL. The requirements are intended to ensure the claimed identity is the actual identity of the person attempting to enroll with the CSP and that scalable attacks effecting a large population of enrolled individuals are difficult to execute without significant time and cost.
+This section lists the steps a CSP SHALL follow to identity proof an individual to meet the requirements for each IAL. The requirements are intended to ensure the claimed identity is the actual identity of the subject attempting to enroll with the CSP and that scalable attacks affecting a large population of enrolled individuals require a greater time and cost than the value of the resources the system is protecting.
 
 ## <a name="resolve"></a>5.1. Identity Resolution
+
+The goal of identity resolution is to uniquely distinquish an individual among a given population or context.  Effective identity resolution uses the smallest set of attribtues necessary to resolve to a unique individual. It provides the CSP an important starting point in the overall identity proofing process, to include the initial detection of potential fraud, but in no way represents a complete and successful identity proofing transaction.  
```

Comment:
I use Markdown Pro (Mac). I have had a few problems with it crashing randomly but it journals everything frequently so haven't lost work as a result. But haven't found anything better.